### PR TITLE
GHA zip workflow: add manual trigger

### PR DESCRIPTION
**Pull request by maksymmalyhin**
An attempt to add a [manual trigger](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) for zip GHA workflow. 

I was not able to test the trigger itself as the "Run workflow" button didn't appear for me. I guess the change has to land to the default branch to be picked up by GHA.

### CODE REVIEW

**Comment by maksymmalyhin** [comment]
The updated package step triggered the correct command for non-manual run: https://github.com/firebase/firebase-ios-sdk/runs/1826829858?check_suite_focus=true#step:4:12
Further packaging was cancelled.

File: .github/workflows/zip.yml [hunk 1]
```diff
@@ -13,6 +13,13 @@ on:
     # Run every day at midnight(PST) - cron uses UTC times
     - cron:  '0 8 * * *'
 
+  workflow_dispatch:
+    inputs:
+      custom_spec_repos:
+        description: 'Custom Podspec repos'
+        required: true
+        default: 'https://cpdc-internal.git.corp.google.com/firebase.git'
```
Comment (paulb777) [inline .github/workflows/zip.yml:21]: I don't think GHA has access to `https://cpdc-internal.git.corp.google.com`. I believe we need the whole staging process to SpecsStaging or another public repo.

File: .github/workflows/zip.yml [hunk 2]
```diff
@@ -37,7 +44,7 @@ jobs:
     - name: ZipBuildingTest
       run: |
          mkdir -p zip_output_dir
```
Comment (paulb777) [inline .github/workflows/zip.yml:46]: Add an `xcode-select` command to Xcode 12.0

File: .github/workflows/zip.yml (hunk 1 above)
Comment (maksymmalyhin) [reply .github/workflows/zip.yml:21]: Yeah, I would prefer moving release staging to GitHub to simplify GHA access control. I think we can address it in following PRs. For now I would like to test the manual trigger in general.

**Review by paulb777 — APPROVED** [review]

File: .github/workflows/zip.yml (hunk 2 above)
Comment (maksymmalyhin) [reply .github/workflows/zip.yml:46]: I would prefer to add it in a separate PR. With the manual trigger testing will be so much easier :).

**Pull request merged**